### ISSUE TITLE
Descriptive text within social icon links gets flagged in accessibility scans for inability to zoom into text

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Footer-Plugin#38]
**Describe the bug**

See title; the spritesheet method used on these icons results does not meet WCAG 2.1 Success Criterion 1.4.4 - Resize Text according to SiteImprove, due to the overflow + text-indent styling used: https://www.w3.org/WAI/WCAG21/Understanding/resize-text

**Steps to Reproduce**

n/a

**Expected Behavior**

We need to display these icons without getting flagged for this issue.

**Possible Fix**

Consider moving inner text of each link to an `aria-label` attribute instead.

**Your environment:**

n/a

**Additional context**

n/a

**Comment by cjg89** [comment]
Resolved in v1.0.9.
